ios/KaMPKitiOS.xcodeproj/project.pbxproj

```diff
@@ -170,7 +170,7 @@
 				F1465EF923AA94BF0055F7C3 /* Sources */,
```

Comment:
Not sure I understand why anything is changing in here.

Reply:
Xcode chatter